README.md

```diff
@@ -1,32 +1,26 @@
-# tutorials
+# howto
```

Comment:
This name change is harder than I thought. `@datamade/tutorials` isn't _quite_ right, since now we're using it for documenting knowledge in general in addition to specific tutorials. I'd proposed `how-to-datamade` originally, which I still think is cute, but I don't like the repetitiveness of `@datamade/how-to-datamade`. Naming things is hard!

Some other ideas:

- `@datamade/howto` (shown here)
- `@datamade/knowhow`
- `@datamade/knowledge`
- `@datamade/stack`

Any thoughts?

Reply:
how about `the-way-the-truth-the-light`? 

jk, i like `how-to` (hyphenated) or `stack`. `stack` feels slightly more rigid, which i like... but if we are planning to use this to document practices beyond our toolkit, then i think i like `how-to` best. 

seems like that's your top choice, to. shall we go with it?!

Reply:
Great, let's go with `how-to` then!